src/containers.js

```diff
@@ -0,0 +1,142 @@
+import {bytesToString, toUint8} from './byte-helpers.js';
+
+export const getId3Offset = function(bytes) {
+  bytes = toUint8(bytes);
+
+  if (bytesToString(bytes.subarray(0, 3)) !== 'ID3') {
+    return 0;
+  }
+  const returnSize = (bytes[6] << 21) |
+                     (bytes[7] << 14) |
+                     (bytes[8] << 7) |
+                     (bytes[9]);
+  const flags = bytes[5];
+  const footerPresent = (flags & 16) >> 4;
+
+  if (footerPresent) {
+    return returnSize + 20;
+  }
+  return returnSize + 10;
+};
+
+export const isLikely = {
+  aac(bytes) {
+    const offset = getId3Offset(bytes);
+
+    return bytes.length >= offset + 2 &&
+      (bytes[offset] & 0xFF) === 0xFF &&
+      (bytes[offset + 1] & 0xE0) === 0xE0 &&
+      (bytes[offset + 1] & 0x16) === 0x10;
+  },
+
+  mp3(bytes) {
```

Comment:
Do we want to consider mp3 without an ID3v2 container?

Reply:
Same question for aac above. Or do they always have an id3 container but may not include any data in it?

Reply:
We consider mp3/aac without id3. `getId3Offset` returns `0` if it cannot find any `id3` data at the start of the file. Which means we will start checking to see if the file is `aac/mp3` at the start of the file.